packages/send/backend/prisma/migrations/20250415170530_uuid_for_users_and_containers/migration.sql

```diff
@@ -0,0 +1,54 @@
+/*
+  Warnings:
+
+  - The primary key for the `Container` table will be changed. If it partially fails, the table could be left without primary key constraint.
+  - A unique constraint covering the columns `[id]` on the table `Container` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[id]` on the table `User` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropForeignKey
+ALTER TABLE "Container" DROP CONSTRAINT "Container_parentId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Item" DROP CONSTRAINT "Item_containerId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Share" DROP CONSTRAINT "Share_containerId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "_ContainerToTag" DROP CONSTRAINT "_ContainerToTag_A_fkey";
+
+-- AlterTable
+ALTER TABLE "Container" DROP CONSTRAINT "Container_pkey",
+ALTER COLUMN "id" DROP DEFAULT,
+ALTER COLUMN "id" SET DATA TYPE TEXT,
+ALTER COLUMN "parentId" SET DATA TYPE TEXT,
+ADD CONSTRAINT "Container_pkey" PRIMARY KEY ("id");
+DROP SEQUENCE "Container_id_seq";
+
+-- AlterTable
+ALTER TABLE "Item" ALTER COLUMN "containerId" SET DATA TYPE TEXT;
```

Comment:
Are we using postgres? If so I think they have a uuid field type. It should have some optimizations under the hood to allow better indexing than text afaik.